pontos/git/git.py

```diff
@@ -0,0 +1,199 @@
+        """
+        Init a git repository
+
+        Args:
+            bare: Wether to create a `bare` repository or not.
+                  Defaults to false.
+        """
+        args = ["init"]
+        if bare:
+            args.append("--bare")
+        _exec_git(*args, cwd=self._cwd)
+
+    def create_branch(self, branch: str, *, start_point: Optional[str] = None):
+        """
+        Create a new branch
+
+        Args:
+            branch: Name of the branch to be created
+            start_point: An optional git reference (branch, tag, sha, ...) from
+                         where to start the branch
+        """
+        args = ["checkout", "-b", branch]
+        if start_point:
+            args.append(start_point)
+
+        _exec_git(*args, cwd=self._cwd)
+
+    def rebase(
+        self,
+        base: str,
+        *,
+        head: Optional[str] = None,
+        onto: Optional[str] = None,
+    ):
+        """
+        Rebase a branch
+
+        Args:
+            base: Apply changes of this branch
+            head: Apply changes on this branch. If not set the current branch is
+                  used.
+            onto: Apply changes on top of this branch
+        """
+        args = ["rebase"]
+
+        if onto:
+            args.extend(["--onto", onto])
+
+        args.append(base)
+
+        if head:
+            args.append(head)
+
+        _exec_git(*args, cwd=self._cwd)
+
+    def clone(
+        self,
+        repo_url: str,
+        destination: Path,
+        *,
+        branch: Optional[str] = None,
+        remote: Optional[str] = None,
+    ):
+        """
+        Clone a repository
+
+        Args:
+            repo_url: URL of the repo to clone
+            destination: Where to checkout the clone
+            branch: Branch to checkout. By default the default branch is used.
+            remote: Store repo url under this remote name
+        """
+        args = ["clone"]
+        if remote:
+            args.extend(["-o", remote])
+        if branch:
+            args.extend(["-b", branch])
+        args.extend([repo_url, str(destination.absolute())])
+
+        _exec_git(
+            *args,
+            cwd=self._cwd,
+        )
+
+    def push(
+        self, *, remote: Optional[str] = None, branch: Optional[str] = None
+    ):
+        """
+        Push changes to remote repository
+
+        Args:
+            remote: Push changes to the named remote
+            branch: Branch to push. Will only be considered in combination with
+                    a remote.
+        """
+        args = ["push"]
+        if remote:
+            args.append(remote)
+            if branch:
+                args.append(branch)
+
+        _exec_git(*args, cwd=self._cwd)
+
+    def config(self, key: str, value: str):
+        """
+        Set a (local) git config
+        """
+        _exec_git("config", key, value, cwd=self._cwd)
+
+    def cherry_pick(self, commit: str):
```

Comment:
```suggestion
    def cherry_pick(self, commit: List[str]):
```

I would love this to be a `List[str]`, so multiple commits can be cherry-picked!